build.sbt

```diff
@@ -355,13 +350,16 @@ def sharedNativeImageSettings(targetDir: File) = Seq(
   "-H:+AllowVMInspection",
   "-H:-RuntimeAssertions",
   "-H:+ReportExceptionStackTraces",
+  "-H:-TraceClassInitialization", // if "+" prints out all trace information about class initialization
   "-H:-PrintUniverse", // if "+" prints out all classes which are included
   "-H:-NativeArchitecture", // if "+" Compiles the native image to customize to the local CPU arch
   "-H:Class=" + "io.cloudstate.proxy.CloudStateProxyMain",
   "--verbose",
   //"--no-server", // Uncomment to not use the native-image build server, to avoid potential cache problems with builds
-  //"--report-unsupported-elements-at-runtime", // Hopefully a self-explanatory flag
+  //"--debug-attach=5005", // Debugger makes a ton of sense to use to debug SubstrateVM
+  "--report-unsupported-elements-at-runtime", // Hopefully a self-explanatory flag
```

Comment:
We should get rid of this when we can—right now it is needed to pass the native-image generation, but I would rather have things surfacing at build time.